projects/igniteui-angular/src/lib/grids/grid/grid-filtering-ui.spec.ts

```diff
@@ -6500,3 +6552,13 @@ function clickElemAndBlur(clickElem, blurElem) {
     UIInteractions.simulatePointerEvent('pointerup', clickElem.nativeElement, elementRect.left, elementRect.top);
     UIInteractions.simulateMouseEvent('click', clickElem.nativeElement, 10, 10);
 }
+
+function clickHtmlElemAndBlur(clickElem, blurElem) {
+    const elementRect = clickElem.getBoundingClientRect();
+    UIInteractions.simulatePointerEvent('pointerdown', clickElem, elementRect.left, elementRect.top);
+    blurElem.nativeElement.blur();
+    clickElem.focus();
+    blurElem.nativeElement.dispatchEvent(new FocusEvent('focusout', { bubbles: true }));
+    UIInteractions.simulatePointerEvent('pointerup', clickElem, elementRect.left, elementRect.top);
+    UIInteractions.simulateMouseEvent('click', clickElem, 10, 10);
+}
```

Comment:
Add the types to the method parameters. Add `DebugElement` types to the `clickElemAndBlur` method and make it call the `clickHtmlElemAndBlur` to avoid code duplication.